src/tbp/monty/frameworks/loggers/graph_matching_loggers.py

```diff
@@ -255,6 +260,14 @@ def update_overall_stats(self, mode, episode, episode_steps, monty_matching_step
             )
             stats["monty_steps"].append(episode_steps)
             stats["monty_matching_steps"].append(monty_matching_steps)
+            # older LMs don't have prediction error stats
+            if "episode_avg_prediction_error" in episode_stats.keys():
```

Comment:
This shouldn't need the `keys()` call. Using `in` on a dictionary checks for whether the value is a key in the dictionary.
```suggestion
            if "episode_avg_prediction_error" in episode_stats:
```